tsconfig.json

```diff
@@ -19,12 +19,12 @@
 		"noUnusedParameters": false,
 		"noUnusedLocals": false,
 
-		"sourceMap": true
+		"sourceMap": true,
 
-	},
-
-	"files": [
```

Comment:
No idea what I have configured differently then because all the settings are applied.